Makefile

```diff
@@ -76,6 +76,11 @@ LOGDNA_HOST?=localhost:1337
 
 RUST_LOG?=info
 
+space := $(subst ,, )
+comma := ,
+FEATURES?=
+FEATURES_ARG=$(if $(FEATURES),--features $(subst $(space),$(comma),$(FEATURES)))
```

Comment:
Nice way to get around quote escaping!

Reply:
The first version I had used a function that was called everywhere, Took me a while to remember = was lazy in Make